languages/purescript/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasagna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `expectedMinutesInOven` function that does not take any input and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
+
+```purescript
+expectedMinutesInOven :: Int
+expectedMinutesInOven
```

Comment:
I'm not sure I follow why the `=` and right side are missing in these. Is it because they're supposed to add it? displaying invalid syntax in examples seems dangerous for beginners.